code/modules/mob/living/silicon/robot/dogborg/dog_sleeper_vr.dm

```diff
@@ -671,4 +671,12 @@
 	desc = "A mounted matter decompiling unit with fuel processor."
 	icon_state = "decompiler"
 	max_item_count = 10
-	decompiler = TRUE
\ No newline at end of file
+	decompiler = TRUE
+
+// PLACEHOLDER SERVICE BORG BELLY
+/obj/item/device/dogborg/sleeper/servborg
+	name = "Gourmet Gut"
+	desc = "An underslung microwave, with optional fuel processor. (WIP)"
```

Comment:
If you really want to take a step further you don't need to add (WIP) to the end of it. Unless you wanna call it an early prototype ingame or something.

"An underslung microwave, with an optional fuel processor attatched. It seems to be an early prototype and might not function completely as intended..."

Reply:
I removed the "(WIP)" from the description - the belly acts like a brigbelly, for now, so I was doing that to let people who use it know it isn't a microwave, yet :P